{{ cookiecutter.project_name }}/.pre-commit-config.yaml

```diff
@@ -0,0 +1,19 @@
+default_language_version:
```

Comment:
Een precommit-hook: ik snap de lol ervan. Maar ik heb ook het gevaar gezien. 
- Het moet idealiter ook bij adviseurs werken.
- Een library, die wordt nog wel eens een jaartje of twee met rust gelaten. Werkt dan alles nog? Ik heb wel knokpartijen gehad voordat ik fixes erin kon krijgen.

Reply:
pre-commit hoef je niet te gebruiken: als je niet `pre-commit install` hebt gedaan, gebeurt er niets!

Enne, libraries die een jaartje of twee met rust zijn gelaten? Maar draai je de unittests dan wel nog in de goede (moderne) environment? Ik vind dat spannend. Goede keuze in deze cookiecutter om de "tests" mee te shippen, dan kan je het in ieder geval nog in de applicatie doen waar de library is gebruikt.